pennylane/__init__.py

```diff
@@ -180,13 +178,30 @@ def __getattr__(name):
     if name == "plugin_devices":
         return pennylane.devices.device_constructor.plugin_devices
 
+    import warnings  # pylint: disable=redefined-outer-name, import-outside-toplevel
```

Comment:
Could we just do the following to get rid of the first pylint error? And then update the warnings below to match

```suggestion
    from warnings import warn  # pylint: disable=import-outside-toplevel
```

Reply:
changed locally.